### DISCUSSION TITLE
Is it possible for exporting a Godot game as an Android Livewallpaper?

### DISCUSSION THREAD

**Discussion by jane00** [godotengine/godot-proposals#6541] [Platforms]
### Describe the project you are working on

Simple game for use as android live wallpaper run in background.

### Describe the problem or limitation you are having in your project

In the templet and source code, I can only find "https://godotforums.org/d/20875-live-wallpaper-and-godot-help" this thread. No solution possible.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Give another export as Android service, not only rendering in activity.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

I don't know how to do this.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No.

### Is there a reason why this should be core and not an add-on in the asset library?

I think both is OK, core or add-on. I am not familiar with Godot and Android

**Comment by TheOathMan** [comment]
Yes. I made an android plugin that let you run your Godot app as a live wallpaper [here](https://github.com/TheOathMan/Godot-Android-Live-Wallpaper). The Plugin is compatible with Godot 4.2.2.stable

**Reply by jane00** [reply]
Greate！！
